cli/src/main/java/org/infinispan/cli/commands/Shutdown.java

```diff
@@ -55,19 +52,12 @@ public boolean isHelp() {
 
       @Override
       public CommandResult exec(ContextAwareCommandInvocation invocation) {
-         if (help) {
-            invocation.println(invocation.getHelpInfo());
-         }
-         CommandInputLine cmd = new CommandInputLine(Shutdown.CMD)
-               .arg(TYPE, Shutdown.Server.CMD)
-               .arg(SERVERS, servers);
-         return invocation.execute(cmd);
+         return invocation.call(c -> servers.isEmpty() ? c.server().stop() : c.cluster().stop(servers));
```

Comment:
it needs a null check (and include the description from Wolf's PR so I can close it eh eh: https://github.com/infinispan/infinispan/pull/8911)
```
[pedro-desktop-21448@cluster//containers]> shutdown server
null
```

Reply:
Done